### PR TITLE
Clarify Install Location

### LINKED ISSUE #1827: No download link to Docker Machine for Linux
I've been trying to look at https://docs.docker.com/machine/ https://docs.docker.com/machine/get-started/ and https://docs.docker.com/linux/started/ and can't find a link to Download Docker Machine for Linux.

Is Docker Machine only available for OS X and Linux?

### PR DESCRIPTION
**Pull request by ehazlett**
This adds a link to the release page and calls out the install section.

Closes #1827

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ managing them. Using these commands you can
  - upgrade the Docker client and daemon
  - configure a Docker client to talk to your host
 
+## Install
```
Comment (moxiegirl) [inline docs/index.md]: I would tend to leave the heading off all together as the sentence would come at the end of the bullet list in the intro section --- it gives enough emphasis.  If you don't like that, I"d tweak the heading level and the head itself so as not to imply these are instructions which the command 

`#### Need the installation instructions?`

**Comment by nathanleclaire** [comment]
LGTM

**Comment by nathanleclaire** [comment]
Thanks for doing this

**Comment by dmp42** [comment]
Maybe a word about using homebrew? wdyt?

**Comment by moxiegirl** [comment]
@ehazlett Are we advising them to use brew?  If we want them to use brew we should update the installation docs --- this is just a pointer about where to find them.

**Comment by dmp42** [comment]
I would assume most reasonable people on OSX would pick a brew solution first.
If we are happy with brew formula quality, I would point it as the main solution - getting binaries manually as an alternative.

**Comment by moxiegirl** [comment]
@dmp42 On OSX, we are actually expecting most people to start off with Toolbox which installs Machine and many other things including Kitematic.

On the Engine side, we seemed to be discouraging tools like `brew` in favor of our installation mechanisms.  This is because things like brew and other similar package managers weren't being kept up-to-date.

**Comment by nathanleclaire** [comment]
> This is because things like brew and other similar package managers weren't being kept up-to-date.

This would be my concern with recommending `brew` as well.

**Comment by dmp42** [comment]
Expecting people to go with Toolbox as a first choice is certainly fine, and is already the suggestion here. This discussion is about the second choice (binaries + manual finickying vs. brew)

About homebrew being up to date, they seem to be doing a good job at job: https://github.com/Homebrew/homebrew/commits/master/Library/Formula/docker-machine.rb

**Comment by moxiegirl** [comment]
Is there a reason we just wouldn't recommend curl for OSX which is what the other Linux platforms use.  Omitting brew from our documentation doesn't prevent people from using it.  It **does** simplify our recommendations and keep us internally consistent with our other projects.

**Comment by nathanleclaire** [comment]
Yeah, I agree with @moxiegirl -- I think we risk inconsistency if we recommend brew, and shouldn't do it unless we're willing to commit to officially supporting it (i.e. we maintain the `brew` recipe directly).

**Comment by dmp42** [comment]
> Is there a reason we just wouldn't recommend curl for OSX which is what the other Linux platforms use. 

Overall: 

 * dangerous (no verification of any sort)
 * cumbersome (manual change of permissions)
 * anti-friendly as far as upgrading/maintenance is concerned

There is even a "wall of shame" for softwares recommending `curl | sh` (what we do here is basically the same thing): http://curlpipesh.tumblr.com/

I certainly agree there is some level of endorsement if we are going to recommend using brew - but at the very least we should stop detailing steps that are counter to best practices (curling without any verification, using `/usr/local/bin` on OSX for eg).

2 cents.

**Comment by nathanleclaire** [comment]
I agree with all of your concerns @dmp42 , but we followed Compose's lead on this one: https://github.com/docker/compose/releases (although they do have an optional installation through a package manager (pip) as well).  I think that before either project changes the recommendation, we need:

1. Be agreed upon by both projects that this is now the recommended install, and present it in a uniform way.
2. An assigned team member who is clearly responsible for updating the brew recipes, otherwise there is a risk that it will not get done adequately.

**Comment by nathanleclaire** [comment]
After discussion with @dmp42 we've decided to move forward with this and we can follow-up discussion and further changes in new PRs.  Thanks @ehazlett @moxiegirl

**Comment by moxiegirl** [comment]
@nathanleclaire this actually didn't have my docs LGTM.  I'll carry the changes I asked for in another PR.

**Pull request merged**